exporters/prometheus/src/test/java/io/opentelemetry/exporter/prometheus/PrometheusHttpServerTest.java

```diff
@@ -318,7 +320,7 @@ private static List<MetricData> generateTestData() {
             InstrumentationScopeInfo.builder("grpc").setVersion("version").build(),
             "grpc.name",
             "long_description",
-            "1",
```

Comment:
A unit of `1` is a poor choice for a sum. Updating to `unit` to have more realistic test data.